config/webpack/plugin/SetupPlugin.js

```diff
@@ -15,13 +46,92 @@ class SetupPlugin {
     const production = compiler.options.mode === 'production';
     const processPath = process.cwd();
 
+    // MODULE
+    // Remove raw markdown rule if found.
+    const mdIndex = compiler.options.module.rules.findIndex((rule) => rule.test.toString && rule.test.toString() === '/\\.md$/');
+    if (mdIndex > -1) {
+      compiler.options.module.rules.splice(mdIndex, 1);
+    }
+
+    const mdxLoader = getMdxLoader(this.publicPath);
+
+    compiler.options.module.rules.push({
+      test: /\.mdx$/,
+      use: [
+        babelLoader,
+        mdxLoader,
+      ],
+    }, {
+      test: /\.md$/,
+      oneOf: [
+        {
+          // To preserve previous behaviour, we will only apply mdx to requests issued
+          // from either the content config files or from .md or .mdx files
```

Comment:
Can you explain this more? I'm confusing & don't see where  content config files is documented at

Reply:
Prior to now based on terra-toolkit's webpack config, `.md` files were loaded as raw files. I'm changing that behavior based on the issuer of the require request. If the file requiring the `.md` file is either a `.mdx` file (which was previously unsupported) or the generated `contentConfig.js` file located in dev-site/config then we will run the `.md` component through the mdx loader, otherwise we will load it as a raw file to not break existing pages.